src/features/theme/darkTheme.ts

```diff
@@ -0,0 +1,32 @@
+import { createMuiTheme, Theme } from '@material-ui/core/styles'
+
+export function darkTheme(theme: Theme){
+    return createMuiTheme({
+        ...theme,
+        palette: {
+            ...theme.palette,
+            type: 'dark', 
+            primary: {
+                main: '#757575',
```

Comment:
why primary and secondary colors are the same? it's weird...

Reply:
This required by material-table customize. It use secondary color for checkbox

Reply:
but it doesn't mean that primary and secondary colors should be the same, do it? I think primary and secondary colors should be different, you just need make sure that checkbox looks ok with secondary color

or, another option, you can try to override checkbox component so that it uses primary color rather than secondary one (if you need it)

Reply:
Remove secondary color from theme and add customize for table filter row color